content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,266 @@
+---
+reviewers:
+- erictune
+title: Init Contêineres
```

Comment:
How would you feel about using "contêineres de inicialização"?

Reply:
I'm correcting one by one ok!